torchtune/datasets/multimodal/_llava_instruct.py

```diff
@@ -89,22 +94,35 @@ def __call__(self, sample: Mapping[str, Any]) -> Mapping[str, Any]:
                     role="system", content=self.new_system_prompt, masked=True, eot=True
                 )
             )
+
+        # Add in image stuffs / load from file
         for message in sample[self._column_map["conversations"]]:
             role = role_map[message["from"]]
+            content = message["value"]
             if role == "system" and self.new_system_prompt is not None:
                 continue
-            content = split_text_by_image_tag(message["value"], "<image>")
+            if role == "user":
+                image_path = sample[self._column_map["image"]]
```

Comment:
So this will be present in every sample in the dataset?

Reply:
yup